lib/ruby_lsp/requests/document_symbol.rb

```diff
@@ -108,6 +108,17 @@ def visit_class(node)
 
       sig { override.params(node: SyntaxTree::Command).void }
       def visit_command(node)
+        if node.message.value == "test"
```

Comment:
Since we're making this change, how about including other common names as well?:

- describe
- context
- it

They can still appear in `minitest`-based tests I believe.

Reply:
I do want to add support for that soon, but it gets a little more complex since they can be nested.

Reply:
Yep, I had those in mind for a follow-up.